app/routes.js

```diff
@@ -62,6 +63,17 @@ export const routes = (authProvider) => {
     res.render('pages/organisation/template.njk', addContext({ context, user: req.user }));
   }));
 
+  router.post('/organisations/:organisationId', authProvider.authorise(), withCatch(authProvider, async (req) => {
```

Comment:
should this route be POST `/organisations/:organisationId/edit` or are you going to do this later?
We do the same for the `adduser` the get and the post have `/adduser`

Reply:
next ticket